scripts/exgdas_atmos_analysis_wdqms.sh

```diff
@@ -1,6 +1,28 @@
 #!/bin/bash
 set -x
 
+################################################################################
+# UNIX Script Documentation Block
+# Name:     exgdas_atmos_analysis_wdqms.sh
+# Author:   Rahul Mahajan
+# Org:      NCEP/EMC
+# Abstract: This script unpacks GSI diagnostic files, runs them through a python
+#           script wdqms.py and creates CSV files for certain observation sub-types
```

Comment:
@aerorahul  Thanks for adding the documentation to the WDQMS script.
How about we add some information about what is included in the CSV files.
Basically, the information included is the following:
- observation information (metadata and observation values)
- residual statistics 
- data usage information in data assimilation

```
This script unpacks GSI diagnostic files, runs them through a Python script, wdqms.py, and creates CVS files. These files include observation information, residual statistics, and data usage in the analysis for certain observation sub-types
```

Reply:
done.